scaife_viewer/views.py

```diff
@@ -302,13 +302,16 @@ def search_json(request):
 
 
 def morpheus(request):
-    if "word" not in request.GET:
-        raise Http404()
+    if ("word" not in request.GET) or ("lang" not in request.GET):
+        raise Http400()
     word = request.GET["word"]
+    lang = request.GET["lang"]
+    if lang not in ["grc","lat"]:
```

Comment:
minor thing (we still need to hook up flake8 CI to catch this on PR) but can you put a space after comma?